docs/content/advanced/backend/batch-jobs/create.md

```diff
@@ -251,6 +254,9 @@ The first step is to create a batch job using the [Create Batch Job endpoint](ht
 
 For example, this creates a batch job of the type `publish-products`:
 
+<Tabs groupId="request-types">
+<TabItem value="client" label="Medusa JS Client" default>
+
 ```jsx
 // using JS Client
```

Comment:
There's a line similar to this for all blocks can you remove them? For JS Client, Fetch API, and cURL code snippets.

Reply:
If I understood you correctly, you would like me to remove the blank line?

Reply:
No I meant the `//using JS Client` line. Every code block for JS Client, Fetch API, and cURL have a line at the beginning similar to this one. They should be removed. Sorry if it wasn't clear.